src/ui/common/package.json

```diff
@@ -37,7 +37,8 @@
     "@fortawesome/free-brands-svg-icons": "^6.1.1",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/react-fontawesome": "^0.1.18",
-    "@mui/lab": "*",
+    "@mui/base": "^5.0.0-alpha.84",
```

Comment:
@agiron123 this is the change I made to get my env working, so that versions here matches versions in `app`